subprojects/platform-base/src/main/java/org/gradle/api/plugins/internal/DefaultBasePluginExtension.java

```diff
@@ -55,35 +55,41 @@ public Property<String> getArchivesName() {
 
 
     @Override
+    @Deprecated
```

Comment:
❓ I thought the plan was no more new deprecations for Gradle 8.0 so that upgraders don't see a bunch of new warnings, just to make notes of things to deprecate in 8.1?

Reply:
The superclass already deprecates these. This just adds the annotations in the child class to make the build happy. `:plugins` ignores deprecation usages so moving this code into `:platform-base` throws new warnings now that we're overriding deprecated methods and not ignoring the warnings